e2e/steps/test_undo_redo.py

```diff
@@ -0,0 +1,60 @@
+from e2e.conftest import prepare_planting, plant_a_tomato, suffix
+from e2e.pages.home import HomePage
+from e2e.pages.login import LoginPage
+from e2e.pages.maps.create import MapCreatePage
+from e2e.pages.maps.planting import MapPlantingPage
+from e2e.pages.maps.management import MapManagementPage
+from pytest_bdd import scenarios, given, when, then, parsers
+
+scenarios("features/undo_redo.feature")
+
+
+@given(parsers.parse("I am on the {name} map page and I have selected the plant layer"))
+def on_planting_screen(
+    hp: HomePage,
+    lp: LoginPage,
+    mmp: MapManagementPage,
+    mcp: MapCreatePage,
+    mpp: MapPlantingPage,
+    name,
+):
+    prepare_planting(hp, lp, mmp, mcp, mpp, name + suffix())
+
+
+# Scenario 1: Successful undo
+
+
+@when("I plant something")
+def plant_something(mpp: MapPlantingPage):
+    plant_a_tomato(mpp)
+
+
+@when("I press undo")
+def press_undo(mpp: MapPlantingPage):
+    mpp.click_undo()
+    mpp.click_on_canvas()
```

Comment:
Doesn't this belong to the step "my plant is gone"?

Reply:
Good point, ty.